src/utils/spo.ts

```diff
@@ -600,10 +600,10 @@ export const spo = {
  * @param webUrl Web url
  * @param email The email of the user
  * @param logger the Logger object
- * @param debug set if debug logging should be logged 
+ * @param verbose set for verbose logging
  */
-  async getUserByEmail(webUrl: string, email: string, logger: Logger, debug?: boolean): Promise<any> {
-    if (debug) {
+  async getUserByEmail(webUrl: string, email: string, logger?: Logger, verbose?: boolean): Promise<any> {
```

Comment:
Why are `logger` and `verbose` optional? In the references, these parameters appear to always be assigned.

Reply:
Correct, they are indeed always assigned. The reason I chose to make these optional is to keep the util functions uniform within the cli. I looked here at how other existing util functions handled the `verbose` and `logger` parameters and they were always passed as optional. Let me know if you want them to be mandatory in this case